dev-docs/bidders/admixerwl.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: bidder
+title: AdmixerWL
+description: Prebid AdMixer Bidder Adaptor
+pbjs: true
+biddercode: admixerwl
+aliasCode: admixer
+media_types: banner, video, native
```

Comment:
please add `multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid` meta data to explain what's happening for multi format ad units

Reply:
Added this metadata.